app/models/user.rb

```diff
@@ -264,6 +267,8 @@ def visible_collections_and_document_sets(user)
     # document set show_to? logic:
     #     (!self.restricted && self.works.present?) || (user && user.like_owner?(self)) || (user && user.collaborator?(self))
     public_collections = self.unrestricted_collections
+    blocked_collection_ids = CollectionBlock.where(user_id: current_user.id).pluck(:collection_id)
```

Comment:
`current_user.id` should probably be `user.id` since that's passed in as a parameter.